flask_app/models.py

```diff
@@ -527,6 +527,7 @@ class Error(db.Model, TypenameMixin):
     timestamp = db.Column(db.Float, default=get_current_time)
     is_failure = db.Column(db.Boolean, default=False)
     is_interruption = db.Column(db.Boolean, default=False)
+    is_fatal = db.Column(db.Boolean)
```

Comment:
Continuing my above comment, `is_fatal` should be `nullable=True`

Reply:
it's true by default

Reply:
This raises another important note - the REST API should not return NULLs for this field as it is an implementation detail. We need to make sure after the migration, the API returns it as `false`